docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+bot:
+  build: .
+  volumes:
+    - .:/web
+  ports:
+    - "8800:80"
+  command: /bin/bash -c "cd /web/;
```

Comment:
Why you need pip install again, it already in dockerfile, i suggest use image `python:onbuild` instead of custom `Dockerfile`